x-pack/plugins/apm/server/lib/transactions/charts/get_anomaly_data/fetcher.ts

```diff
@@ -91,7 +74,8 @@ export async function anomalySeriesFetcher({
   };
 
   try {
-    return await client<void, Aggs>('search', params);
+    const response = await client<void, typeof params>('search', params);
+    return response;
```

Comment:
it's kind of weird to me that this is in a try/catch and silently fails when there is a HttpError. Anyone who knows why?